python/cuml/test/test_coordinate_descent.py

```diff
@@ -89,6 +89,43 @@ def test_lasso(datatype, X_type, alpha, algorithm,
         assert array_equal(sk_predict, cu_predict, 1e-1, with_sign=True)
 
 
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+@pytest.mark.parametrize('X_type', ['ndarray'])
+def test_lasso_default(datatype, X_type):
+    train_rows = np.int32(20*0.8)
+    X, y = make_regression(n_samples=20, n_features=3,
+                           n_informative=2, random_state=0)
+    X_test = np.asarray(X[train_rows:, 0:], dtype=datatype)
+    X_train = np.asarray(X[0:train_rows, :], dtype=datatype)
+    y_train = np.asarray(y[0:train_rows, ], dtype=datatype)
+
+    cu_lasso = cuLasso()
```

Comment:
Did two different changes get mixed together here?

Reply:
Yes. This PR shouldn't be merged until after Saloni's PR (or my commits merged with hers before merging her PR).